src/pages/IndivPlaylistPage.js

```diff
@@ -40,23 +39,23 @@ function IndivPlaylistPage () {
         }
     }
 
-    async function findAndAddSong(song) {
-        const songTitle = song.val;
-        try {
-            const result = await axios.get("/auth/search/" + songTitle);
-            if (result !== undefined) {
-                const songToAdd = result.data["tracks"]["items"][0];
-                updateList(songToAdd);
-            }
-            else {
-                console.log("Not found.");
-            }
-        }
-        catch(error) {
-            console.log("I'M A FAILURE");
-            console.log(error);
-        }
-    }
+    // async function findAndAddSong(song) {
+    //     const songTitle = song.val;
+    //     try {
+    //         const result = await axios.get("/auth/search/" + songTitle);
+    //         if (result !== undefined) {
+    //             const songToAdd = result.data["tracks"]["items"][0];
+    //             updateList(songToAdd);
+    //         }
+    //         else {
+    //             console.log("Not found.");
+    //         }
+    //     }
+    //     catch(error) {
+    //         console.log("I'M A FAILURE");
+    //         console.log(error);
+    //     }
+    // }

```

Comment:
Why did this get commented out? Everything inside the function call seems like it could work: from the axios call to the result indexing...

Reply:
The function works, but it was no longer necessary in this page. It was only commented out in case we might want to use it in the future.